training/watch.md

```diff
@@ -0,0 +1,29 @@
+---
+---
+
+# 3. When do you watch spam?
+Adding websites and keywords to be watched by SmokeDetector is a big part of being a privileged
+user on Charcoal. Unless a post is ***obvious*** spam, it can sometimes be a bit overwhelming to
+keep up with new software terms, libraries and projects (nuget, chocolatey, mustache, etc.) and
+know what is legit and what is not. We recommend doing a bit of research before you add a watch.
+
+ - **Search** for the URL or keyword on [Stack Exchange](https://stackexchange.com/)
+   - Use `url:example.com` in the search bar to find all questions and answers that have that website
+     in them. Check how many hits come up and if it's only the spam and 2-3 other posts it will
+     typically be OK to add it.
+   - Keywords can be searched just like normal text, but multiple keywords should be enclosed in
+     quotation marks. Example: `"key word"` and `"more words to search for"`. In this way it will
+     only show results for the actual text you search for. Stack Exchange will otherwise show results
+     for each searched word and this will typically return too many hits.
+ - **Check** [open pull requests](https://github.com/Charcoal-SE/SmokeDetector/pulls) on SmokeDetector
+   Github. The watch you want to add might already be queued for approval. It's always a good idea to
+   check and make sure it's not a duplicate.
+
```

Comment:
This is badly missing another bullet point worth of advice to search MS for your proposed watch. This would also be a good place to link/include @makyen's bookmarklet for doing an MS search.

Reply:
Added section on bookmarklets and searching on MS